modules/setup/classes/UserInfo.php

```diff
@@ -0,0 +1,259 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: root
```

Comment:
snuff these comments please.

Reply:
yeah i am on kali @aethelwulffe and my default user is root reason why ech time i creat a file ,class etc phpstorm (my editor) puts it by default i will change that later cause i guess i have many files to create still hope i dont forget.